card/lib/card/active_record_helper.rb

```diff
@@ -5,8 +5,9 @@ def create_card args
       Card.create! args
     end
 
+    # if card with same name exists move it out of the way
     def create_card! args
-      create_card args.reverse_merge(rename_if_conflict: :new)
+      create_card args.reverse_merge(rename_if_conflict: :old)
```

Comment:
hm, but it creates bad names. I changed it because I got something like "Formula 1" on Wikirate because there was a useless "formula" basic card in the db.
I think in a deck card migration it's likely that you want the card exactly as in your migration (there is already a ! in the method name as a warning) but in a core card migration you are maybe right that it's better to mess less with the deck cards of other people.